packages/tsconfig.base.json

```diff
@@ -2,15 +2,24 @@
   "compilerOptions": {
     "jsx": "react",
     "lib": [
-      "es2017",
+      "es2015",
       "dom"
     ],
-    "esModuleInterop": true,
-    "module": "commonjs",
+    "target": "es2015",
+    "module": "es2015",
     "moduleResolution": "node",
+    "esModuleInterop": true,
     "skipLibCheck": true,
+    "importHelpers": true,
+    "composite": true,
+    "sourceMap": true,
+    "declarationMap": true,
```

Comment:
Do we need to also add `"declaration": true`? Or does the declarationDir setting in the individual package configs maybe take care of generating these?

Reply:
By default `tsc` emits declarations. [This is the flag we were using before:](https://www.typescriptlang.org/docs/handbook/compiler-options.html)

Option | Type | Default | Description
-- | -- | -- | --
--noEmit | boolean | false | Do not emit outputs.